packages/flutter/test/cupertino/text_selection_toolbar_test.dart

```diff
@@ -274,6 +274,52 @@ void main() {
     expect(findOverflowBackButton(), findsNothing);
   }, skip: kIsWeb); // [intended] We do not use Flutter-rendered context menu on the Web.
 
+  testWidgetsWithLeakTracking('correctly sizes large toolbar buttons', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      CupertinoApp(
+        home: Center(
+          child: SizedBox(
+            width: 420,
+            child: CupertinoTextSelectionToolbar(
+              anchorAbove: const Offset(50.0, 100.0),
+              anchorBelow: const Offset(50.0, 200.0),
+              children: const <Widget>[
+                SizedBox(width: 100),
+                SizedBox(width: 300),
+                SizedBox(width: 100),
+                SizedBox(width: 100),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+
+    Finder findSizedBoxWithWidth(double width) =>
+      find.byWidgetPredicate((Widget widget) => widget is SizedBox && widget.width == width);
```

Comment:
I think you should remove the first `=>` based on https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#consider-using--for-short-functions-and-methods.